### ISSUE TITLE
Export Notebook to EPUB

### ISSUE THREAD

**Issue by infosisio** [notable/notable#525]
This should be trivial and could probably be more handy than a PDF, due to its accessibility issues.

**Comment by fabiospampinato** [comment]
Why do you say it should be trivial? Is there a nice standalone library for doing this? 

Exporting to PDF was easy because that capability is built-in into Electron, the framework Notable is based on. In order to export notes to EPUB we would have to convert them somehow. There's [pandoc](https://pandoc.org/) for these conversion needs, but that's quite heavy, we can't just bundle it with the app.

**Comment by infosisio** [comment]
@fabiospampinato EPUB is just compressed html + toc/xml. You don't really need much to generate it. But it still may not be a good idea to go this far and include such functionality. If it ever becomes a good idea though, it's useful.

**Comment by lmihalkovic** [comment]
Supporting the spines and some of the other nice things is not exactly ‘trivial’ (wrote a epub editor in objc once)

**Comment by fabiospampinato** [comment]
If anyone knows of a good NPM package for converting HTML to EPUB I can add support for this easily.
